cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1587,4 +1587,10 @@
     </description>
   </property>
 
+  <!-- Dataset extensions -->
+  <property>
+    <name>dataset.extensions.dir</name>
+    <value>/opt/cdap/ext/lib</value>
+  </property>
+
```

Comment:
Rather than adding this to the end of the file, please place this as an additional property between the existing ``dataset.executor.max.instances`` and ``dataset.service.bind.address`` with a ``description`` attribute such as
````
Directory where all dataset extensions are stored
````
This will then automatically be added by the documentation build system to the table generated in the docs, appearing at http://docs.cask.co/cdap/3.2.1/en/admin-manual/installation/cdap-site.html#appendix-cdap-site-xml